silk/hw/hw_module.py

```diff
@@ -29,6 +29,7 @@
 interfaceSerial = 'InterfaceSerialNumber'
 interfaceNumber = 'USBInterfaceNumber'
 dutSerialNumber = 'DutSerial'
+otnsVisLocation = 'OTNSVisLocation'
```

Comment:
```suggestion
otnsVisLocation = 'OTNSVisPosition'
```

`position` is a more accurate word for cordinates?